rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -814,7 +821,7 @@ public int compareTo(AbstractRecord other) {
         return 0;
     }
 
-    public URI getRecoveryCoordinatorURI() {
+    public URI getRecoveryURI() {
```

Comment:
I am wondering what the need for renaming this method was?

Reply:
just because the method name was misleading. It is actually returning `recoveryURI` which is the participant ID not the recovery coordinator URI (which is included in the `recoveryURI` but it's not it).

Reply:
Ah OK, thanks for confirming. I agree it is misleading to include coordinator in the method - please can you add it to a separate commit/JBTM. I think the use of it [here](https://github.com/jbosstm/narayana/blob/cc46def573a1a0692bd531aaa0cc39c7a4b52a41/rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java#L429) should have the assigned variable name altered too. /cc @mmusgrov

Reply:
sure, I will add a new commit and JBTM.

Reply:
Thanks!

Reply:
moved to https://issues.redhat.com/browse/JBTM-3240 as I will probably close this PR depending on the resolution of https://issues.redhat.com/browse/JBTM-3239

Reply:
Thanks @xstefank - if you continue in here please rebase on master now that https://github.com/jbosstm/narayana/pull/1543 is merged